### DISCUSSION TITLE
Windows portable app? Installation path

### DISCUSSION THREAD

**Discussion by frps** [standardnotes/forum#2518] [Suggestions]
It seems the program use at least three different folders to store the app and data.

c:\Users\username\AppData\Local\Programs\standard-notes
c:\Users\username\AppData\Roaming\Standard Notes
C:\Users\username\Standard Notes Backups

The questions are
1) How to install a program to another disk/path? 
2) How to change the database location folder?
3) How to make a portable version like ThunderbirdPortable for example?
https://portableapps.duckduckgo.com/ThunderbirdPortable_60.0_English.paf.exe
AppFolder
- App
- Data
- Other
StandardNotesPortable.exe

This way you can put a program to separate Data disk, external USB disk/stick, encrypted volume and so on.

**Comment by atmoio** [comment]
The first two aren't currently possible, and a portable version is something I've considered but haven't put any time into yet. But definitely interesting and important ideas, just a matter of getting there.

**Comment by AdKiller** [comment]
Any update on this?
Thanks

**Comment by alterdaemon** [comment]
I'll start using (paid)  Standard Notes as soon as portable version is available. Never before.

**Comment by JaspalSuri** [comment]
Thanks for your feedback! I've made a note of this for the team to review.

**Comment by EmadFathy** [comment]
+1 for portable app.

**Comment by nowindowsnogates** [comment]
+1 for portable app.

**Comment by mohitkumarcool** [comment]
+1 for portable app

**Comment by SullieLore** [comment]
+1
